content/docs/started/workstation/getting-started-windows.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Kubeflow on Windows"
+description = "Install Kubeflow on Windows"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+        more quickly
+  TODO: Surface the windows specific instructions here. For instance, when WSL2 is
+        available more broadly, add instructions here.
+-->
+
```

Comment:
Add an introductory sentence as a nice lead-in to this page, as you did in the Linux guide.

Reply:
Done, and I've added a comment about WSL 2